doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst

```diff
@@ -48,6 +48,7 @@ Step 1: Launch the Demo and Configure the Plugin
   * Make sure the **Robot Description** field is set to ``robot_description``.
 
   * Make sure the **Planning Scene Topic** field is set to ``/planning_scene``.
+    (Click on topic name to expose topic-name drop-down.)
```

Comment:
No need for parenthesis here

Reply:
Missed this comment the first time.  Pushed again.